README.md

```diff
@@ -23,3 +24,4 @@ Fizz
 19
 Buzz
 ... etc up to 100
```

Comment:
Can you move this line out of the format?

Reply:
good idea.  done.